ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/spanning-tree.cfg

```diff
@@ -5,6 +5,7 @@ transceiver qsfp default-mode 4x10G
 hostname spanning-tree
 !
 spanning-tree edge-port bpduguard default
```

Comment:
Still seeing diferences on generated configuration on 4.25.5M and AVD generated configuration:
```
spanning-tree mode mstp
no spanning-tree vlan-id 105,202,505-506
spanning-tree mst pvst border
spanning-tree edge-port bpduguard default
spanning-tree edge-port bpdufilter default
spanning-tree mst 0 priority 4096
spanning-tree mst 100-200 priority 8192
!
spanning-tree mst configuration
   name test
   revision 5
   instance 2 vlan  16-18
   instance 3 vlan  15
   instance 4 vlan  200-300
```

Reply:
Spanning-tree mode mstp is already there previously. I didn't change the order. I added logic only for bpdufilter which should be below bpduguard.